sdk/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
     implementation "androidx.security:security-crypto:1.1.0-alpha02"
     implementation "androidx.browser:browser:1.2.0"
     implementation "androidx.appcompat:appcompat:1.2.0"
+    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
```

Comment:
https://developer.android.com/jetpack/androidx/releases/lifecycle

It says the API is deprecated.

Reply:
added dependencies for the specific Lifecycle artifacts (view model)